app/helpers/render_constraints_helper_behavior.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module RenderConstraintsHelperBehavior
+  include Blacklight::RenderConstraintsHelperBehavior
+
+  def remove_constraint_url(localized_params)
+    scope = localized_params.delete(:route_set) || self
+
+    unless localized_params.is_a? ActionController::Parameters
+      localized_params = ActionController::Parameters.new(localized_params)
+    end
+
+    options = localized_params.merge(q: nil, utf8: nil, action: article_search? ? 'new' : 'index')
```

Comment:
@jkeck that sounds promising. I just rebased and the only remaining override is the `action` ternary. Article search's current index action returns this with Blacklight's default remove url query params (`?search_field=search&utf8=✓`):
![screen shot 2017-06-30 at 12 16 46 pm](https://user-images.githubusercontent.com/5402927/27750885-a3f20580-5d8e-11e7-8b8d-410bbdd66a1c.png)

I think I found the helper you're talking about in [catalog/index.html.erb](https://github.com/sul-dlss/SearchWorks/blob/master/app/views/catalog/index.html.erb). I'll give it a shot and ping you when I have something.